pkg/thanos/operator.go

```diff
@@ -17,6 +17,8 @@ package thanos
 import (
 	"context"
 	"fmt"
+	"github.com/prometheus-operator/prometheus-operator/pkg/assets"
+	"github.com/prometheus-operator/prometheus-operator/pkg/webconfig"
```

Comment:
the `github.com/prometheus-operator/prometheus-operator/...`  imports need to go with their siblings.